io.github.kaschpal.timetable.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id": "io.github.kaschpal.timetable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "timetable",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-documents"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+    	"python3-python-dateutil.json",
+    	"python3-mallard-ducktype.json",
+        {
+            "name": "timetable",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/kaschpal/timetable"
```

Comment:
This has to be a specific commit or a tagged release

Reply:
In the sense of changing (--ammend) the commit message to something like "release 1.0" and/or give the commit the tag "v1.0" ?

Reply:
reading other projects, i finally understood ...